### PR TITLE
Datahub / Use proxy for showing data in table and map, rely on mime types in protocol if any

### PR DESCRIPTION
**Pull request by jahow**

### CODE REVIEW

File: libs/feature/record/src/lib/data-view-map/data-view-map.component.ts [hunk 1]
```diff
@@ -151,24 +153,28 @@ export class DataViewMapComponent {
...
       const url = getEsriRestDataUrl(link, 'geojson')
       return fromPromise(
-        readDataset(url, 'geojson').then((features) => ({
-          type: MapContextLayerTypeEnum.GEOJSON,
-          data: {
-            type: 'FeatureCollection',
-            features,
-          },
-        }))
+        readDataset(this.proxy.getProxiedUrl(url), 'geojson').then(
```
Comment (fgravin) [inline libs/feature/record/src/lib/data-view-map/data-view-map.component.ts:169]: duplicated code, should the if just about the `url` var set ?

File: libs/feature/record/src/lib/data-view-table/data-view-table.component.ts [hunk 2]
```diff
@@ -111,7 +113,7 @@ export class DataViewTableComponent {
     } else if (this.linkHelper.isEsriRestFeatureServer(link)) {
       const url = getEsriRestDataUrl(link, 'geojson')
       return fromPromise(
-        readDataset(url, 'geojson').then((features) =>
+        readDataset(this.proxy.getProxiedUrl(url), 'geojson').then((features) =>
```
Comment (fgravin) [inline libs/feature/record/src/lib/data-view-table/data-view-table.component.ts:124]: again, i would have done a shared `readDatasetFn` somewhere, it's like for fetching the dataset we have to change the code at 3 or 4 different places.

**Review by fgravin — APPROVED** [review]
Thanks.
Some comments but not related with your work.

File: libs/feature/record/src/lib/data-view-map/data-view-map.component.ts (hunk 1 above)
Comment (jahow) [reply libs/feature/record/src/lib/data-view-map/data-view-map.component.ts:169]: yeah I agree it's bad... sorry, don't have much time now, we'll revisit this later on I guess

File: libs/feature/record/src/lib/data-view-table/data-view-table.component.ts (hunk 2 above)
Comment (jahow) [reply libs/feature/record/src/lib/data-view-table/data-view-table.component.ts:124]: agreed!

**Comment by jahow** [comment]
@fgravin thanks for the review, I don't have time to address the comments now, we will have to revisit this code and also the format detection because there is way too much duplicated logic

**Pull request merged**